_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,152 @@
+
+One of the worst investors of all time is Sir Isaac Newton. I don't think many people would consider him stupid, but he was a stupid investor.<sup>2</sup>
+
+### How does Benjamin Graham Define These Two Types of Investors?
+
+**Defensive investors** are focused on preserving their investment capital first and foremost. Their goal is the limit their downside, not get astronomical returns or be the next Warren Buffett. These are typically indexers or people who buy into mutual funds like Vanguard.
+
+**Enterprising investors** are willing to devote more time to managing their investments and executing on investment ideas. In short, they spend more time analyzing all the aspects of a business and get to know it thoroughly. These are the investors who are trying to make higher returns over longer periods of time. Perhaps not becoming billionaires, but they focus on becoming significantly wealthier than your average joe.
+
+### When is it a Good Point to Become an Investor?
+
+Everyone can be a mix, but taking big risks when you're in debt or living hand to mouth is flirting with disaster. The money used for  investing is money set aside for this purpose after all your other needs have been met and you're safe.
+
+Benjamin Graham doesn't go into a ton of detail of this, but I think it is safe to say that these are reasonable criteria starts to a list, meaning more reasons exist but these were the ones I could think of on a Sunday afternoon:
+
+  1) You have to have **no debt at all**, when I say that I mean: car payments, student loan debt, credit cards, a large mortgage you need to pay off monthly. Debt can be defined as **owing any entity money**.
+
+  2) Make sure you have a stable income source with excess money to use: jobs, side hustles, social security, winnings from lay suites, etc.
+
+  3) Do you have a safety net of between 6 - 12 months of living expenses saved up? When the shit goes down you better be ready.
+
+
+### Why Am I Focusing on Defensive Investors and not talking about Enterprising Investors?
+
+Many people try, and typically fail, to beat the market averages for many reasons, but the reality is that most people think they're above average<sup>3</sup> when in fact that's statistically impossible. Can _YOU_ beat the market consistently for decades? You can try, but it is likely that you will fail due to sheer time required, temperamental and intellectual advantage within the market, and the element of markets becoming more efficient particularly at the large capital end i.e. companies in the S&P 500.
```

Comment:
You can try, but it is likely that you will fail due to sheer time required, temperamental and intellectual advantage within the market, and the element of markets becoming more efficient **--** particularly at the large capital end **,** i.e. companies in the S&P 500.